src/commons/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+
+import Button, {ButtonSize, ButtonStyle} from './Button';
+
+export default {
+    title: 'Example/Button',
+    component: Button,
+  } as ComponentMeta<typeof Button>;
+
+const Template: ComponentStory<typeof Button> = (args) => <Button
```

Comment:
Let's use props here
```suggestion
const Template: ComponentStory<typeof Button> = (props) => <Button
```